docs/topics/api-clients.md

```diff
@@ -1,527 +0,0 @@
-# API Clients
-
```

Comment:
I have a question, should we remove everything from here? or should we redo this part by removing core api parts only and directing to the new suggestions? or rewriting the examples with new suggestions or we already got that covered in another section?

Reply:
I thought about that too, but then noticed that basically the whole page is about Core API. 

At first I thought about only saving the client part, but then discovered that the whole ecosystem has been stale for 5 years and the repos are all archived. Furthermore, I tried to follow the instructions in `api-clients.md`, but it did not work for me anymore.

regarding rewriting examples: not sure if that makes sense honestly. We wouldn't want to duplicate openapi-generator or drf-spectacular docs here. I wouldn't know what to replace it with.

Reply:
> I have a question, should we remove everything from here?

I think so, yes.

Reply:
@tomchristie with your approval I would merge this and move on to the next task.